features/steps/browser_steps.rb

```diff
@@ -15,8 +17,13 @@
   wait = Selenium::WebDriver::Wait.new(timeout: n)
   wait.until {
     $driver.find_element(id: 'bugsnag-test-state') &&
-    $driver.find_element(id: 'bugsnag-test-state').text == 'DONE'
+    (
+      $driver.find_element(id: 'bugsnag-test-state').text == 'DONE' ||
+      $driver.find_element(id: 'bugsnag-test-state').text == 'ERROR'
+    )
   }
+  txt = $driver.find_element(id: 'bugsnag-test-state').text
+  assert_equal('DONE', txt, "Expected #bugsnag-test-state text to be 'DONE'. It was '#{txt}'.")
```

Comment:
This adds a way for the test fixture app to communicate that it is complete, but that an error happened.